src/aatoolbox/datasources/codab/codab.py

```diff
@@ -75,7 +76,15 @@ def download(self, clobber: bool = False) -> Path:
             clobber=clobber,
         )
 
-    def load(self, admin_level: int) -> gpd.GeoDataFrame:
+    def process(self, *args, **kwargs):
+        """
+        Process COD AB data.
+
+        Method not implemented.
+        """
+        logger.info("`process()` method not implemented for CodAB.")
```

Comment:
nit: if it's not required I would lean towards modifying the log info to explicitly say it )rather than not implemented which seems to imply that it will be implemented at some point)

Reply:
Yes agreed! (but in this case we would like to implement a processing step in the future)